app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_type, file_name, path_with_file = command.split()
+    split_path = path_with_file.split("/")
+    new_file_name = split_path[-1]
+    primary_path = os.getcwd()
+    new_path = "/".join(split_path[:-1:])
+
+    with open(file_name, "r") as prim_file:
```

Comment:
```suggestion
    with open(file_name, "r") as file:
```
better name it as `file` or as `f`